test/studies/shootout/mandelbrot/ldelaney/mandelbrot-release-dynamic.chpl

```diff
@@ -0,0 +1,68 @@
+/* The Computer Language Benchmarks Game
+   http://benchmarksgame.alioth.debian.org/
+
+   contributed by Jacob Nelson, Lydia Duncan, Brad Chamberlain,
+   and Ben Harshbarger
+   derived from the GNU C version by Greg Buchholz
+*/
+
+use DynamicIters;
+
+config const n = 200,              // image size in pixels (n x n)
+             maxIter = 50,         // max # of iterations per pixel
+             limit = 4.0,          // per-pixel convergence limit
+             chunkSize = 1;        // dynamic iterator's chunk size
+
+param bitsPerElt = 8;              // # of bits to store per array element
+type eltType = uint(bitsPerElt);   // element type used to store the image
+
+
+proc main() {
+  const ydim = 0..#n,                          // the image's y dimension
+        xdim = 0..#divceilpos(n, bitsPerElt);  // the compacted x dimension
+
+  var image : [ydim, xdim] eltType;            // the compacted bitmap image
+
+    forall (y, xelt) in dynamic({ydim, xdim}, chunkSize) {
```

Comment:
In the above, I think we should move away from declaring xdim and ydim, simply declare image as:

    var image: [0..#n, 0..#divceilpos(...)];

and then make the forall w/ dynamic be over image.domain

If the high bound for ydim seems too complex to jam into an array declaration, maybe we could just symbolically name its upper bound?